src/magic_folder/test/test_upload.py

```diff
@@ -170,7 +167,7 @@ def test_commit_a_file(self, mangled_name, content, upload_dircap):
         # test whether we got a capability
         self.assertThat(
             remote_snapshot_cap,
-            MatchesPredicate(is_uri,
+            MatchesPredicate(is_immutable_directory_cap,
                              "%r is not a Tahoe-LAFS URI"),
```

Comment:
minor, but the text should maybe be upgraded too (e.g. "not a Tahoe-LAFS immutable-directory URI")